Source/WebCore/dom/BoundaryPoint.cpp

```diff
@@ -64,40 +66,38 @@ static bool isOffsetBeforeChild(ContainerNode& container, unsigned offset, Node&
     return false;
 }
 
-// FIXME: Once we move to C++20, replace with the C++20 <=> operator.
```

Comment:
Why are you removing this comment?

Reply:
Not intentional. I had initially removed these functions and their comment. I had to reintroduce them to fix the build and failed to reintroduce their associated comment